staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/connection_test.go

```diff
@@ -159,7 +159,7 @@ func TestWithValidThumbprint(t *testing.T) {
 func TestWithInvalidCaCertPath(t *testing.T) {
 	connection := &vclib.VSphereConnection{
 		Hostname: "should-not-matter",
-		Port:     "should-not-matter",
```

Comment:
This could have been a fixture too may be. But I agree its part of "doesn't" matter. Lets fix this for not.